include/swift/SIL/SILBasicBlock.h

```diff
@@ -558,17 +558,22 @@ struct PhiOperand {
 
   bool operator!=(PhiOperand other) const { return !(*this == other); }
 
-  BranchInst *getBranch() const {
-    return cast<BranchInst>(predBlock->getTerminator());
+  // TODO: This should eventually be limited to BranchInst.
+  TermInst *getBranch() const {
+    return predBlock->getTerminator();
```

Comment:
The problem with these abstractions is that we still don't have all the necessary SIL restrictions. In particular, OSSA still supports CondBr arguments for trivial types. We deferred doing this work because OSSA utilities generally ignores trivial values anyway. Until we fix this, \p PhiOperand might be invalid for trivial phis.

I started to add support for CondBr arguments everywhere, but reconsidered. I just changed the `getBranch` method back to returning `BranchInst` as before.